### PR TITLE
[MEMBERS] model - migration - seeder - OT102-88

### PR DESCRIPTION
**Pull request by WernerBudtke**
# [OT102-88](https://alkemy-labs.atlassian.net/browse/OT102-88)

# Description:
AS: administrator user
I WANT: to manage the members of the ONG
TO: modify the Members of the Organization

# Acceptance criteria: 
It will be possible to manage the members of the ONG. Name of the table: members. Fields: name, image

# How to reproduce this change:
apply migration:
`npx sequelize-cli db:migrate`
seed:
`npx sequelize-cli db:seed:all`
undo migration:
`npx sequelize-cli db:migrate:undo:all --to XXXXXXXXXXXXXX-nombre-migracion.js`

### CODE REVIEW

File: migrations/20211123063059-create-member.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    await queryInterface.createTable('Members', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
+      },
```
Comment (joselo85) [inline migrations/20211123063059-create-member.js:14]: Al campo de name, le pondria: allowNull: false

File: models/member.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    image: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    deletedAt: DataTypes.DATE
+  }, {
+    sequelize,
+    modelName: 'Member',
+  });
```
Comment (joselo85) [inline models/member.js:35]: Fijate que  hay q agregar lo siguiente: createdAt: 'created_at', updatedAt: 'updated_at', deletedAt: 'deletedAt',  paranoid: true,  timestamps: true,

File: seeders/20211123063059-create-members.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkInsert('Members', [{
+      name: 'John Doe',
+      image: 'https://www.designevo.com/res/templates/thumb_small/colorful-hand-and-warm-community.png',
+      createdAt: new Date,
```
Comment (dvelezroman) [inline seeders/20211123063059-create-members.js]: `createdAt: new Date()`

File: seeders/20211123063059-create-members.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      updatedAt: new Date
```
Comment (dvelezroman) [inline seeders/20211123063059-create-members.js]: `updatedAt: new Date()`

**Review by dvelezroman — CHANGES_REQUESTED** [review]
buen PR.. solo pequenios cambios solicitados

File: package-lock.json [hunk 5]
```diff
@@ -1,8660 +1,8 @@
 {
```
Comment (gabyMangoni) [inline package-lock.json]: Ojo... esto no deberias mergearlo con develop.. a mi me paso lo mismo...

**Review by dvelezroman — APPROVED** [review]
Realizar los cambios y mergear...

**Review by JonatanGarbuyo — APPROVED** [review]

**Review by joselo85 — DISMISSED** [review]

**Review by joselo85 — APPROVED** [review]

**Pull request merged**